internal/backends/mcu/stm32h735g.rs

```diff
@@ -90,7 +103,7 @@ pub fn init() {
 
     let hyperram_size = 16 * 1024 * 1024; // 16 MByte
     let config = hal::xspi::HyperbusConfig::new(80.MHz())
-        .device_size_bytes(24) // 16 Mbyte
+        .device_size_bytes(21) // 16 Mbyte
```

Comment:
The value and the comment seem out of sync. Are you sure it’s only 2MB on your board?

I was under the impression that it’s 16MB and for example https://github.com/STMicroelectronics/STM32CubeH7/blob/c94252df7cec24a8fef67b28933353476a1edd3c/Projects/STM32H735G-DK/Examples/OSPI/OSPI_HyperRAM_MemoryMapped/Inc/main.h#L32 also uses 24.

Reply:
This was not intentional. Thanks for catching.